vncviewer/Viewport.cxx

```diff
@@ -754,6 +754,11 @@ void Viewport::handleClipboardChange(int source, void *data)
     return;
 #endif
 
+  if (!Fl::clipboard_contains(Fl::clipboard_plain_text)) {
+    vlog.debug("Got non-plain text in local clipboard, ignoring.");
+    return;
```

Comment:
Hmm... This is before any of the calls to `announceClipboard()`. So I think we also need to call `announceClipboard(false)`, in case we already had the clipboard and just lost it.

Reply:
Good point. Fixed in 6f6d9406035d59b6a9f59ff0f38206c0c98a0266